__tests__/app.test.js

```diff
@@ -26,3 +26,44 @@ describe('GET /api', () => {
       });
   });
 });
+
+describe('GET /api/topics', () => {
+  test('200: Responds with an array of topics, each with a slug and description', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then(({ body: { result } }) => {
+        expect(Array.isArray(result)).toBe(true);
```

Comment:
result is not a good name for this key, instead I would recommend naming it something that better reflects the value it holds. Something similar to `topics`